src/hostapi/coreaudio/pa_mac_core.c

```diff
@@ -447,11 +447,11 @@ static void DumpDeviceProperties( AudioDeviceID macCoreDeviceId,
 {
     PaError err;
     int i;
-    UInt32 propSize;
-    UInt32 deviceLatency;
-    UInt32 streamLatency;
-    UInt32 bufferFrames;
-    UInt32 safetyOffset;
+    UInt32 propSize = 0;
+    UInt32 deviceLatency = 0;
+    UInt32 streamLatency = 0;
+    UInt32 bufferFrames = 0;
+    UInt32 safetyOffset = 0;
     AudioStreamID streamIDs[128];
```

Comment:
Please move the initialization just prior to use, as is already done for `propSize`

For example, add `bufferFrames = 0;` after line 459

Reply:
This was marked "resolved" but the code was not changed!